app/views/admin/settings/projects_settings/show.html.erb

```diff
@@ -63,7 +54,17 @@ See COPYRIGHT and LICENSE files for more details.
           .reject { |_, options| !options[:caption] }
           .map { |name, options| [options[:caption], name.to_s, checked: options[:builtin], disabled: options[:builtin]] }
       %>
-
+      <div class="form--field">
```

Comment:
This results in the `from--field` being visible all the time, even if there is no banner. Please move it one line below so that it is caught by the `unless` block.